public/components/metrics/view/__tests__/__snapshots__/metrics_grid.test.tsx.snap

```diff
@@ -733,10 +733,10 @@ exports[`Metrics Grid Component renders Metrics Grid Component 1`] = `
               >
                 <Component
                   editMode={false}
-                  fromTime="now-30m"
+                  fromTime="now-1d"
                   key="Y4muP4QBiaYaSxpXk7r8"
                   onEditClick={[MockFunction]}
-                  onRefresh={true}
+                  onRefresh={0}
```

Comment:
is this expected - default from 30m to 1d? Also why is onRefresh turning into false here, is that expected?